practices/practice-two/src/styles/sections/better-search-template.css

```diff
@@ -0,0 +1,109 @@
+.template-wrapper {
+  background-color: var(--sixth-color);
+}
+
+.template-item {
+  display: grid;
+  gap: 30px;
+}
+
+.template-image {
+  background-color: transparent;
+  position: relative;
+}
+
+.image {
+  width: 508px;
+}
+
+.image-absolute {
+  position: absolute;
+}
+
+.template-list-tick {
+  padding: 20px 0;
+}
+
+/* TODO: Add to late */
+/*
+mobile devices - sm
+ */
+@media screen and (min-width: 320px) and (max-width: 480px) {
+  .image {
+    width: 100%;
+    transform: translateX(-50px);
+  }
+
+  .image-absolute {
+    width: 125px;
+    top: 27px;
+    left: 36.5%;
+  }
+}
+
+/*
+iPads & tablets - md
+*/
+@media screen and (min-width: 481px) and (max-width: 768px) {
+  .image {
+    transform: translateX(-75px);
+  }
+  .image-absolute {
+    top: 45px;
+    left: 40.5%;
+  }
+}
+
+/*
+styles common mobile and ipads: c-md
+*/
+@media screen and (min-width: 320px) and (max-width: 768px) {
+  .template-item {
+    grid-template-columns: repeat(1, 1fr);
```

Comment:
grid-template-columns: 1fr;

Reply:
Because I want to do that to fix the number of columns I want to display for convenience